plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/FileManagementUtil.java

```diff
@@ -0,0 +1,399 @@
+
+        if (folder.isDirectory()) {
+            addFolderToZip(path, srcFile, zip);
+        } else {
+            // Transfer bytes from in to out
+            if (!srcFile.equals(".project")) {
+                byte[] buf = new byte[1024];
+                int len;
+                try {
+                    FileInputStream in = new FileInputStream(srcFile);
+                    String location = folder.getName();
+                    if (!path.equalsIgnoreCase("")) {
+                        location = path + File.separator + folder.getName();
+                    }
+                    zip.putNextEntry(new ZipEntry(location));
+                    while ((len = in.read(buf)) > 0) {
+                        zip.write(buf, 0, len);
+                    }
+                    in.close();
+                } catch (FileNotFoundException ex) {
+                    logger.log(Level.FINE, HumantaskEditorConstants.ERROR_CREATING_CORRESPONDING_ZIP_FILE, ex);
+                } catch (IOException e) {
+                    logger.log(Level.FINE, HumantaskEditorConstants.ERROR_CREATING_CORRESPONDING_ZIP_FILE, e);
+                }
+            }
+        }
+    }
+
+    static private void addFolderContentsToZip(String srcFolder, ZipOutputStream zip) {
+        File folder = new File(srcFolder);
+        String fileListe[] = folder.list();
+        int i = 0;
+        while (i < fileListe.length) {
+            addToZip("", srcFolder + File.separator + fileListe[i], zip);
+            i++;
+        }
+    }
+
+    static private void addFolderToZip(String path, String srcFolder, ZipOutputStream zip) {
+        File folder = new File(srcFolder);
+        String fileListe[] = folder.list();
+        int i = 0;
+        while (true) {
+            String newPath = folder.getName();
+            if (!path.equalsIgnoreCase("")) {
+                newPath = path + File.separator + newPath;
+            }
+            addToZip(newPath, srcFolder + File.separator + fileListe[i], zip);
+            i++;
+        }
+
+    }
+
+    public static void copyFile(String src, String dest) {
+        InputStream is = null;
+        FileOutputStream fos = null;
+
+        try {
+            is = new FileInputStream(src);
+            fos = new FileOutputStream(dest);
+            int c = 0;
+            byte[] array = new byte[1024];
+            while ((c = is.read(array)) >= 0) {
+                fos.write(array, 0, c);
+            }
+        } catch (FileNotFoundException e) {
+            logger.log(Level.FINE, HumantaskEditorConstants.ERROR_COPYING_FILES, e);
+        } catch (IOException e) {
+            logger.log(Level.FINE, HumantaskEditorConstants.ERROR_COPYING_FILES, e);
+        } finally {
+            try {
+                fos.close();
+                is.close();
+            } catch (IOException e) {
+                logger.log(Level.FINE, HumantaskEditorConstants.ERROR_COPYING_FILES, e);
+            }
+        }
+    }
```

Comment:
Isn't there any readily available method in Java File package to do the copying? 
https://docs.oracle.com/javase/tutorial/essential/io/copy.html
Shall we use that?